alacritty/src/renderer/mod.rs

```diff
@@ -378,8 +380,9 @@ impl GlyphCache {
         let padding_y = f64::from(window_config.padding.y) * dpr;
 
         // Calculate new size based on cols/lines specified in config.
-        let grid_width = cell_width as u32 * dimensions.columns_u32();
-        let grid_height = cell_height as u32 * dimensions.lines_u32();
+        // Override cols/lines if they are lower than the limit.
```

Comment:
This comment doesn't really document the right thing. It uses imperative, but the statements here do not exist to serve this exact purpose.

The two sentences should likely be combined to make sense as a whole, rather than just adding one thing to another in an incohesive list.